src/main/resources/db_scripts/delta_script_v0_to_v1.sql

```diff
@@ -0,0 +1,45 @@
+alter table "job_instance"
+  rename column "event_id"
+  to "sensor_event_id";
+
+alter table "event_trigger"
+  RENAME TO sensor;
+
+alter table "sensor"
+  rename column "event_type"
+  to "sensor_type";
+
+delete from "event";
+
+ALTER TABLE "event"
+  ADD COLUMN "sensor_id" BIGINT NOT NULL;
+
+alter table "job_instance"
+  add constraint "job_instance_job_definition_fk"
+  foreign key("job_definition_id")
+  references "job_definition"("id")
+  on update NO ACTION on delete NO ACTION;
+
+alter table "job_instance"
+  add constraint "job_instance_sensor_event_fk"
+  foreign key("sensor_event_id")
+  references "event"("sensor_event_id")
```

Comment:
I am not sure how this works, but this line does not follow standards other references in this PR are.

Reply:
Point of this pr is to set convention for DB scripts and delta scripts.

Reply:
Another issue #27